docs/wiki/get-started/Installation.md

```diff
@@ -1,20 +1,29 @@
-## `pip`
+## Pre-requisites
 
-We ship binary wheels to install CSP on MacOS and Linux via `pip`:
+On most machines, you need Python >=3.8 on your machine to install csp.
+On ARM MacOS, you need Python >=3.10  to install csp.
```

Comment:
this is not true, we just need to update our build (but sdist build will work on ARM MAC)